packages/flutter/test/material/dropdown_test.dart

```diff
@@ -70,6 +72,8 @@ Widget buildFrame({
                 child: Text(item, key: ValueKey<String>(item + 'Text')),
               );
             }).toList(),
+            selectedItemBuilder: selectedItemBuilder,
+            itemHeight: itemHeight,
```

Comment:
Good: tests that don't specify itemHeight should work as before since kMinInteractiveDimension is the default.